### PR TITLE
Enables BigTable IT test for various test suites

### PR DESCRIPTION
**Pull request by chamikaramj**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Flink/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_VR_Spark/lastCompletedBuild/)
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Spark_Batch/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Python3_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) <br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Spark/lastCompletedBuild/)

Pre-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

--- |Java | Python | Go | Website
--- | --- | --- | --- | ---
Non-portable | [Build Status](https://builds.apache.org/job/beam_PreCommit_Java_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Go_Cron/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Website_Cron/lastCompletedBuild/) 
Portable | --- | [Build Status](https://builds.apache.org/job/beam_PreCommit_Portable_Python_Cron/lastCompletedBuild/) | --- | ---

See [.test-infra/jenkins/README](https://github.com/apache/beam/blob/master/.test-infra/jenkins/README.md) for trigger phrase, status and link of all Jenkins jobs.

### CODE REVIEW

**Comment by chamikaramj** [comment]
Run Python PostCommit

**Comment by chamikaramj** [comment]
cc: @tvalentyn @aaltay

**Comment by chamikaramj** [comment]
Run Python PostCommit

**Comment by chamikaramj** [comment]
Run Python PreCommit

**Comment by chamikaramj** [comment]
Retest this please

File: sdks/python/test-suites/direct/py35/build.gradle [hunk 1]
```diff
@@ -34,6 +34,7 @@ task postCommitIT(dependsOn: 'installGcpTest') {
         "apache_beam.io.gcp.bigquery_io_read_it_test",
         "apache_beam.io.gcp.bigquery_read_it_test",
         "apache_beam.io.gcp.bigquery_write_it_test",
+        "apache_beam.io.gcp.bigtableio_it_test",
```
Comment (tvalentyn) [inline sdks/python/test-suites/direct/py35/build.gradle]: there is also https://github.com/apache/beam/blob/acb1f561cc0803367010dc025d13a41b50611f7d/sdks/python/build.gradle#L108 . The location of Py27 suite is currently not consistent with 3.5-3.7.

**Comment by chamikaramj** [comment]
Retest this please

**Comment by chamikaramj** [comment]
Run Python Postcommit

File: sdks/python/test-suites/direct/py35/build.gradle (hunk 1 above)
Comment (chamikaramj) [reply sdks/python/test-suites/direct/py35/build.gradle]: Done.

**Comment by chamikaramj** [comment]
Run Python Postcommit

**Comment by tvalentyn** [comment]
Run Python Postcommit

**Comment by tvalentyn** [comment]
Run Portable_Python PreCommit

**Comment by tvalentyn** [comment]
Run Python Postcommit

File: sdks/python/apache_beam/io/gcp/bigtableio_it_test.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
 import uuid
 
 import pytz
+from nose.plugins.attrib import attr
```
Comment (tvalentyn) [inline sdks/python/apache_beam/io/gcp/bigtableio_it_test.py]: I think line 68 meant to call superclass of `beam.PTransform` (a copy-paste error from `WriteToBigTable` implementation?). This error is caught in Py3 test suites.
It seems strange that we need to create a new transform to write test data - why is that the case, @chamikaramj ?

Comment (tvalentyn) [reply sdks/python/apache_beam/io/gcp/bigtableio_it_test.py]: The syntax should be:
```super(GenerateTestRows, self).__init__()```

**Comment by tvalentyn** [comment]
What's next on this PR?

**Pull request closed without merging**